packages/documentation/src/pages/getting-started/common-tasks/build.mdx

```diff
@@ -1,25 +0,0 @@
-# Building VA.gov 
```

Comment:
Consolidated this content into the "Run and build" document. Felt that running and building were similar enough to be explained in the same context.